packages/docs/src/examples/ConnectingToNetwork.tsx

```diff
@@ -21,17 +23,54 @@ ReactDOM.render(
 const STAKING_CONTRACT = '0x00000000219ab540356cBB839Cbe05303d7705Fa'
 
 export function App() {
-  const { activateBrowserWallet, deactivate, account } = useEthers()
+  const { account, activateBrowserWallet, deactivate } = useEthers()
   const userBalance = useEtherBalance(account)
   const stakingBalance = useEtherBalance(STAKING_CONTRACT)
+
+  const ConnectButton = () => (
+    <div>
+        <button onClick={() => activateBrowserWallet()}>Connect</button>
+        <p>Connect to wallet to interact with the example.</p>
+    </div>
+  )
+
+  const MetamaskConnect = () => (
```

Comment:
Not using the extracted one?

Reply:
Yes, because i want to show users code single time:)

Reply:
Makes sense.